tests/integration/callbacks/test_basic_callback.py

```diff
@@ -791,3 +791,31 @@ def on_click(_):
     dash_duo.start_server(app)
 
     assert dash_duo.get_logs() == []
+
+
+def test_cbsc019_callback_running(dash_duo):
+    app = Dash(__name__)
+
+    app.layout = html.Div(
+        [
+            html.Div("off", id="running"),
+            html.Button("start", id="start"),
+            html.Div(id="output"),
+        ]
+    )
+
+    @app.callback(
+        Output("output", "children"),
+        Input("start", "n_clicks"),
+        running=[[Output("running", "children"), "on", "off"]],
+        prevent_initial_call=True,
+    )
+    def on_click(_):
+        time.sleep(1.0)
```

Comment:
Rather than a `sleep` can we do this test with a `Lock`, like in [`cbsc013`](https://github.com/plotly/dash/blob/4822b7b61915ce21bf8c0367492e70489fd4d02b/tests/integration/callbacks/test_basic_callback.py#L593-L612) for example? It's both faster and more robust that way.

Reply:
I couldn't make it work with the lock, it updates too fast for the driver.